roboteam_ai/src/utilities/Coach.cpp

```diff
@@ -211,7 +222,8 @@ void Coach::removeDefender(int id) {
     }
 }
 
-Vector2 Coach::getRobotClosestToPosition(std::vector<roboteam_msgs::WorldRobot> &robots, Vector2 position, bool includeSamePosition) {
+Vector2 Coach::getRobotPositionClosestToPositionPosition(std::vector<roboteam_msgs::WorldRobot> &robots,
```

Comment:
This function is very odd and I think we should remove it

Reply:
Will be changed